pathod/log.py

```diff
@@ -65,7 +65,7 @@ def dump(self, data, hexdump):
             for line in netlib.utils.hexdump(data):
                 self("\t%s %s %s" % line)
         else:
-            for i in netlib.utils.clean_bin(data).split("\n"):
+            for i in netlib.utils.clean_bin(data).split(b"\n"):
                 self("\t%s" % i)
```

Comment:
Oh damn, you're right. Thanks for catching this! We then need to decode `i` somehow?